packages/rich-text/src/plugins/Table/createTablePlugin.ts

```diff
@@ -38,7 +45,14 @@ export const createTablePlugin = (): PlatePlugin =>
     withOverrides: (editor, plugin) => {
       const { normalizeNode } = editor;
       // injects important fixes from plate's original table plugin
-      withTable(editor, plugin);
+      editor = withNormalizeTable(editor);
```

Comment:
We reset this on line 59 below. So maybe drop it?

Reply:
done!